src/main/java/com/exasol/containers/ExasolContainer.java

```diff
@@ -596,6 +604,28 @@ private void clusterConfigurationIsAvailable() {
         }
     }
 
+    private static final int MAJOR_DEPRECATED_VERSION = 6;
+    private static final int MINOR_DEPRECATED_VERSION = 2;
+
+    private void ThrowDBVersionNotSupportedException() {
+        throw new ContainerLaunchException(
+                ExaError.messageBuilder("E-ETC-13")
+                        .message("Exasol Database version " + MAJOR_DEPRECATED_VERSION + "." + MINOR_DEPRECATED_VERSION
+                                + " and lower are no longer supported in this version of Exasol Testcontainers.")
+                        .toString());
+    }
+
+    private void CheckClusterConfigurationForMinimumSupportedDBVersion() {
```

Comment:
```suggestion
    private void checkClusterConfigurationForMinimumSupportedDBVersion() {
```

Reply:
+ reorder. Move this one above the other new one.

Reply:
ok